### PR TITLE
docs: Update README.md for badges

### PR DESCRIPTION
**Pull request by that-ar-guy**
fixes: #251

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,13 @@
-# greenstar
+# GreenSTAR
+
+[![Build Status](https://github.com/arikkfir/greenstar/actions/workflows/build.yaml/badge.svg)](https://github.com/arikkfir/greenstar/actions/workflows/)
```
Comment (arikkfir) [inline README.md:3]: This needs updating as that workflow has been renamed to `e2e-tests.yaml`.
I also suggest we add `linter.yaml` and `unit-tests.yaml` as well.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+[![Go Report Card](https://goreportcard.com/badge/github.com/arikkfir/greenstar/backend)](https://goreportcard.com/report/github.com/arikkfir/greenstar/backend)
+![TypeScript](https://img.shields.io/badge/TypeScript-4.x-blue.svg)
```
Comment (arikkfir) [inline README.md:5]: I think this badge is just an icon?

**Review by arikkfir — CHANGES_REQUESTED** [review]
See inline comments, and also could you please force-push the commit, rephrasing it to conform to conventional commits - as laid out in [CONTRIBUTING.md](https://github.com/arikkfir/greenstar/blob/main/CONTRIBUTING.md#conventions)? 🙏

File: README.md (hunk 1 above)
Comment (arikkfir) [inline README.md:3]: This needs updating as that workflow has been renamed to `e2e-tests.yaml`.
I also suggest we add `linter.yaml` and `unit-tests.yaml` as well.

File: README.md (hunk 2 above)
Comment (arikkfir) [inline README.md:5]: I think this badge is just an icon?

**Review by arikkfir — CHANGES_REQUESTED** [review]
(resubmitting review - seems comments were on the wrong lines)

See inline comments, and also could you please force-push the commit, rephrasing it to conform to conventional commits - as laid out in [CONTRIBUTING.md](https://github.com/arikkfir/greenstar/blob/main/CONTRIBUTING.md#conventions)? 🙏

File: README.md (hunk 1 above)
Comment (that-ar-guy) [reply README.md]: fixed changes see commit : 15687b7

**Review by arikkfir — CHANGES_REQUESTED** [review]
Looks better!

The TypeScript badge still looks weird to me - it doesn't mention this repository specifically, so what does it base the grade on?

File: README.md (hunk 2 above)
Comment (that-ar-guy) [reply README.md:7]: Should i just remove the typescript badge then

Comment (arikkfir) [reply README.md:7]: I'm ok with that for now, sure.

Comment (that-ar-guy) [reply README.md:7]: Alright, I am removing it now

**Comment by arikkfir** [comment]
Also, would be great if you can squash your commits so that there's just one commit (so that we drop the first commit which isn't a conventional commit)

File: README.md (hunk 2 above)
Comment (that-ar-guy) [reply README.md]: Resolved changes see commit: 3e94a2c

**Review by arikkfir — CHANGES_REQUESTED** [review]
Much better, thank you 🙏 
Please just squash the commits to a single commit (conventional commit) and we should be good to go.

**Comment by that-ar-guy** [comment]
> Much better, thank you 🙏 Please just squash the commits to a single commit (conventional commit) and we should be good to go.
 
resolved changes in : 6f22094

**Comment by arikkfir** [comment]
> > Much better, thank you 🙏 Please just squash the commits to a single commit (conventional commit) and we should be good to go.
> 
> resolved changes in : [6f22094](https://github.com/arikkfir/greenstar/commit/6f22094bf023fd2ac0877a2c51e7f0f1dd08fc89)

I'm still seeing 6 commits in the PR... ? (admittedly it's commits already present in source repository `main` but with different commit IDs - this would cause conflicts)

You should bring the PR to have just 1 commit being contributed, lmk if you need some help.

**Comment by that-ar-guy** [comment]
> > > Much better, thank you 🙏 Please just squash the commits to a single commit (conventional commit) and we should be good to go.
> > 
> > 
> > resolved changes in : [6f22094](https://github.com/arikkfir/greenstar/commit/6f22094bf023fd2ac0877a2c51e7f0f1dd08fc89)
> 
> I'm still seeing 6 commits in the PR... ? (admittedly it's commits already present in source repository `main` but with different commit IDs - this would cause conflicts)
> 
> You should bring the PR to have just 1 commit being contributed, lmk if you need some help.

changed to single commit : 022de3a

working on conflicts

**Review by arikkfir — APPROVED** [review]
LGTM

**Comment by arikkfir** [comment]
Merged, thanks!

**Pull request merged**